[26 feb 2018 16:29:26,395][INFO ][][] Using a custom ConnectionKeepAliveStrategy with default duration = 'infinite' 
[26 feb 2018 16:29:26,400][INFO ][][] Using a custom ConnectionReuseStrategy based on peer HTTP headers/version 
[26 feb 2018 16:29:26,402][INFO ][][] Using a custom HttpRequestRetryHandler to delegate retry logic to JMS redelivery mechanism 
[26 feb 2018 16:29:26,403][DEBUG][][] Creating shared HTTP client 
[26 feb 2018 16:29:26,403][DEBUG][][] httpsClientProperties.getTrustURLString():[file:///D:/MyGitRepo/Dynatrace2RemedyLocalRepo/parent/dynatrace2RemedyEngine/src/main/resources/truststore.jks] 
[26 feb 2018 16:29:26,449][DEBUG][][] createTrustManagers - Initializing trust manager: dynatracetest 
[26 feb 2018 16:29:26,449][DEBUG][][] Initializing trust manager 
[26 feb 2018 16:29:26,522][DEBUG][][] --------- protocolHandShake:[TLSv1.2]-------- 
[26 feb 2018 16:29:27,120][DEBUG][][] Shared HTTP client created successfully! 
[26 feb 2018 16:29:27,120][INFO ][][] Starting IdleHTTPConnectionEvictor task (evictionPeriod=5000) 
[26 feb 2018 16:29:27,167][INFO ][][] Using a custom ConnectionKeepAliveStrategy with default duration = 'infinite' 
[26 feb 2018 16:29:27,168][INFO ][][] Using a custom ConnectionReuseStrategy based on peer HTTP headers/version 
[26 feb 2018 16:29:27,168][INFO ][][] Using a custom HttpRequestRetryHandler to delegate retry logic to JMS redelivery mechanism 
[26 feb 2018 16:29:27,170][DEBUG][][] Creating shared HTTP client 
[26 feb 2018 16:29:27,170][DEBUG][][] --------- protocolHandShake:[TLSv1.2]-------- 
[26 feb 2018 16:29:27,204][DEBUG][][] Shared HTTP client created successfully! 
[26 feb 2018 16:29:27,204][INFO ][][] Starting IdleHTTPConnectionEvictor task (evictionPeriod=5000) 
[26 feb 2018 16:29:27,280][DEBUG][][] getting the inputStream from fileName:[ADR.properties] 
[26 feb 2018 16:29:27,282][DEBUG][][] propertyValue:[ADBM;TAXI] for propertyName:[DASHBOARD_NAMES] 
[26 feb 2018 16:29:27,283][DEBUG][][] propertyValue:[RemedyUsr] for propertyName:[REMEDY_AUTH_INFO_USER] 
[26 feb 2018 16:29:27,284][DEBUG][][] propertyValue:[RemedyPwd         ] for propertyName:[REMEDY_AUTH_INFO_PWD] 
[26 feb 2018 16:29:27,285][DEBUG][][] propertyValue:[https://dynatrace-sub-CA:8021/rest/management/dashboard/ADBM_TEST_REMEDY_INTEGRATION] for propertyName:[ADBM_URL] 
[26 feb 2018 16:29:27,285][DEBUG][][] propertyValue:[admin] for propertyName:[ADBM_USER] 
[26 feb 2018 16:29:27,285][DEBUG][][] propertyValue:[adradmin01] for propertyName:[ADBM_PWD] 
[26 feb 2018 16:29:27,286][DEBUG][][] propertyValue:[https://dynatrace-sub-ca:8021/rest/management/dashboard/TAXI_TEST_REMEDY_INTEGRATION] for propertyName:[TAXI_URL] 
[26 feb 2018 16:29:27,286][DEBUG][][] propertyValue:[admin] for propertyName:[TAXI_USER] 
[26 feb 2018 16:29:27,286][DEBUG][][] propertyValue:[adradmin01] for propertyName:[TAXI_PWD] 
[26 feb 2018 16:29:27,286][DEBUG][][] Configuration DONE 
[26 feb 2018 16:29:27,346][INFO ][AC1F07675A9427D70001B514][1] Flow RemedyIncidentdBuilder Engine Called... 
[26 feb 2018 16:29:27,347][INFO ][AC1F07675A9427D70001B514][1] service:[it.reply.sytel.adr.services.impl.GetDynatraceIncident] called ...  
[26 feb 2018 16:29:27,347][DEBUG][AC1F07675A9427D70001B514][1] the Name for app:[ADBM] is [ADBM] 
[26 feb 2018 16:29:27,348][DEBUG][AC1F07675A9427D70001B514][1] the URL for app:[ADBM] is [https://dynatrace-sub-CA:8021/rest/management/dashboard/ADBM_TEST_REMEDY_INTEGRATION] 
[26 feb 2018 16:29:27,348][DEBUG][AC1F07675A9427D70001B514][1] the USR for app:[ADBM] is [admin] 
[26 feb 2018 16:29:27,348][DEBUG][AC1F07675A9427D70001B514][1] the PWD for app:[ADBM] is [adradmin01] 
[26 feb 2018 16:29:27,348][DEBUG][AC1F07675A9427D70001B514][1] calling the console:[https://dynatrace-sub-CA:8021/rest/management/dashboard/ADBM_TEST_REMEDY_INTEGRATION] with user[admin] pwd=adradmin01... 
[26 feb 2018 16:29:27,353][DEBUG][AC1F07675A9427D70001B514][1] Executing POST with basic authentication 
[26 feb 2018 16:29:27,384][DEBUG][AC1F07675A9427D70001B514][1] SOAP HTTP request headers: 
[26 feb 2018 16:29:27,389][INFO ][AC1F07675A9427D70001B514][1] 	name=Authorization, value=Basic YWRtaW46YWRyYWRtaW4wMQ== 
[26 feb 2018 16:29:28,054][INFO ][AC1F07675A9427D70001B514][1]  Server certificate 1: 
[26 feb 2018 16:29:28,054][INFO ][AC1F07675A9427D70001B514][1]   Subject DN: CN=Insecure AppMon Demo, OU=Development, O=Dynatrace LLC, L=Waltham, ST=MA, C=US 
[26 feb 2018 16:29:28,054][INFO ][AC1F07675A9427D70001B514][1]   Signature Algorithm: SHA256withRSA 
[26 feb 2018 16:29:28,054][INFO ][AC1F07675A9427D70001B514][1]   Valid from: Tue Mar 07 11:37:35 CET 2017 
[26 feb 2018 16:29:28,055][INFO ][AC1F07675A9427D70001B514][1]   Valid until: Thu Mar 07 11:37:35 CET 2019 
[26 feb 2018 16:29:28,055][INFO ][AC1F07675A9427D70001B514][1]   Issuer: CN=dynatrace-sub-CA, DC=dynatrace, DC=org 
[26 feb 2018 16:29:28,055][INFO ][AC1F07675A9427D70001B514][1]  Server certificate 2: 
[26 feb 2018 16:29:28,055][INFO ][AC1F07675A9427D70001B514][1]   Subject DN: CN=dynatrace-sub-CA, DC=dynatrace, DC=org 
[26 feb 2018 16:29:28,055][INFO ][AC1F07675A9427D70001B514][1]   Signature Algorithm: SHA256withRSA 
[26 feb 2018 16:29:28,056][INFO ][AC1F07675A9427D70001B514][1]   Valid from: Fri Jan 09 15:41:43 CET 2015 
[26 feb 2018 16:29:28,056][INFO ][AC1F07675A9427D70001B514][1]   Valid until: Sat Jan 09 15:51:43 CET 2021 
[26 feb 2018 16:29:28,056][INFO ][AC1F07675A9427D70001B514][1]   Issuer: CN=DYNATRACE-ROOT 
[26 feb 2018 16:29:28,056][INFO ][AC1F07675A9427D70001B514][1]  Server certificate 3: 
[26 feb 2018 16:29:28,057][INFO ][AC1F07675A9427D70001B514][1]   Subject DN: CN=DYNATRACE-ROOT 
[26 feb 2018 16:29:28,057][INFO ][AC1F07675A9427D70001B514][1]   Signature Algorithm: SHA256withRSA 
[26 feb 2018 16:29:28,057][INFO ][AC1F07675A9427D70001B514][1]   Valid from: Thu Jan 08 15:18:45 CET 2015 
[26 feb 2018 16:29:28,057][INFO ][AC1F07675A9427D70001B514][1]   Valid until: Fri Jan 08 15:28:40 CET 2027 
[26 feb 2018 16:29:28,057][INFO ][AC1F07675A9427D70001B514][1]   Issuer: CN=DYNATRACE-ROOT 
[26 feb 2018 16:29:28,919][INFO ][AC1F07675A9427D70001B514][1] Remote peer HTTP version > 1.0 --> keep-alive=true 
[26 feb 2018 16:29:28,920][INFO ][AC1F07675A9427D70001B514][1] Current connection will be kept alive 'indefinitely' 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] SOAP HTTP response headers: 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] 	name=Date, value=Mon, 26 Feb 2018 15:29:28 GMT 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] 	name=Set-Cookie, value=JSESSIONID=obp49t8rahspxb04gina6qm8;Path=/;Secure;HttpOnly 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] 	name=Expires, value=Thu, 01 Jan 1970 00:00:00 GMT 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Disposition, value=inline; filename="ADBM_TEST_REMEDY_INTEGRATION.xml" 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Type, value=text/xml 
[26 feb 2018 16:29:28,923][INFO ][AC1F07675A9427D70001B514][1] 	name=Cache-Control, value=no-cache 
[26 feb 2018 16:29:28,924][INFO ][AC1F07675A9427D70001B514][1] 	name=Cache-Control, value=must-revalidate 
[26 feb 2018 16:29:28,924][INFO ][AC1F07675A9427D70001B514][1] 	name=Cache-Control, value=max-age=0 
[26 feb 2018 16:29:28,924][INFO ][AC1F07675A9427D70001B514][1] 	name=Pragma, value=no-cache 
[26 feb 2018 16:29:28,924][INFO ][AC1F07675A9427D70001B514][1] 	name=Vary, value=Accept-Encoding, User-Agent 
[26 feb 2018 16:29:28,924][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Length, value=4983 
[26 feb 2018 16:29:28,924][INFO ][AC1F07675A9427D70001B514][1] HTTP response code received: 200 
[26 feb 2018 16:29:28,925][DEBUG][AC1F07675A9427D70001B514][1] calling the console:[https://dynatrace-sub-CA:8021/rest/management/dashboard/ADBM_TEST_REMEDY_INTEGRATION] with user[admin] pwd=adradmin01... DONE elapsed:[1577] 
[26 feb 2018 16:29:28,925][DEBUG][AC1F07675A9427D70001B514][1] content:[<?xml version="1.0" encoding="utf-8"?>
<dashboardreport name="ADBM_TEST_REMEDY_INTEGRATION" version="7.0.10.1010" reportdate="2018-02-26T16:29:28.643+01:00" description="">
  <source name="ADBM_TEST">
    <filters summary="last 24 hours">
      <filter>tf:Last24h</filter>
    </filters>
  </source>
  <data>
    <incidentsoverviewdashlet name="Incidents" description="" displaysource="Base">
      <source>
        <filters summary="Incident Rules, last 24 hours">
          <filter>tf:Last24h</filter>
          <filter>if:RULE?Agent Instrumentation Disabledif:RULE?Application Process Out-of-Memoryif:RULE?Application Process Unavailable (unexpected)if:RULE?Application Process Unhealthyif:RULE?Average response time degradedif:RULE?CURRENT_Q_DEPTHif:RULE?Database connection issueif:RULE?Failure rate too highif:RULE?Host Availabilityif:RULE?Host CPU Unhealthyif:RULE?Host Disk Unhealthyif:RULE?Host Memory Unhealthyif:RULE?Host Network Unhealthyif:RULE?JDBC Connection Pool Percent Usedif:RULE?JDBC Connection Pool Waiting Thread Countif:RULE?Potentially stuck Java VM detectedif:RULE?Response time degraded for slow requestsif:RULE?Status of the QMif:RULE?Thread Declared Hungif:RULE?Total GC Collection Time</filter>
        </filters>
      </source>
      <incidentoverviewrecords structuretype="tree">
        <incidentoverviewrecord name="ADBM_TEST (6)">
          <incidentoverviewrecord name="Agent Instrumentation Disabled"></incidentoverviewrecord>
          <incidentoverviewrecord name="Application Process Out-of-Memory"></incidentoverviewrecord>
          <incidentoverviewrecord name="Application Process Unavailable (unexpected)"></incidentoverviewrecord>
          <incidentoverviewrecord name="Application Process Unhealthy"></incidentoverviewrecord>
          <incidentoverviewrecord name="CURRENT_Q_DEPTH (5)">
            <incidentoverviewrecord name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name-&gt;ADBM_QU_ANAGRAFICA] (ARAPPX02_QM_THRESHOULD_10@192.168.175.144) ARAPPX02_QM_THRESHOULD_10 upper bound exceeded" start="2018-02-12T19:44:10.000+01:00" source="ARAPPX02_QM_THRESHOULD_10@192.168.175.144"></incidentoverviewrecord>
            <incidentoverviewrecord name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name-&gt;ADBM_QU_NAS] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T19:53:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"></incidentoverviewrecord>
            <incidentoverviewrecord name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name-&gt;ADBM_QU_UFIS_ADR] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T17:30:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"></incidentoverviewrecord>
            <incidentoverviewrecord name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name-&gt;ADBM_QU_UFIS_TEST] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T17:30:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"></incidentoverviewrecord>
            <incidentoverviewrecord name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name-&gt;TELEX_QU_ADBM] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T14:08:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"></incidentoverviewrecord>
          </incidentoverviewrecord>
          <incidentoverviewrecord name="Database Agent Connection Issue"></incidentoverviewrecord>
          <incidentoverviewrecord name="Failure rate too high"></incidentoverviewrecord>
          <incidentoverviewrecord name="JDBC Connection Pool Percent Used"></incidentoverviewrecord>
          <incidentoverviewrecord name="JDBC Connection Pool Waiting Thread Count"></incidentoverviewrecord>
          <incidentoverviewrecord name="Potentially stuck Java VM detected"></incidentoverviewrecord>
          <incidentoverviewrecord name="Response time degraded"></incidentoverviewrecord>
          <incidentoverviewrecord name="Response time degraded for slow requests"></incidentoverviewrecord>
          <incidentoverviewrecord name="Status of the QM"></incidentoverviewrecord>
          <incidentoverviewrecord name="Thread Declared Hung"></incidentoverviewrecord>
          <incidentoverviewrecord name="Total GC Collection Time (1)">
            <incidentoverviewrecord name="Java Virtual Machine/Total GC Collection Time: Total GC Collection Time &lt;all-applications&gt; (WebSphere_ADBM_TEST[arappx03Cell01-app_adbm_mercuriox2]@mercurio) upper bound exceeded" start="2018-02-26T03:00:10.000+01:00" duration="10000" end="2018-02-26T03:00:20.000+01:00" source="WebSphere_ADBM_TEST[arappx03Cell01-app_adbm_mercuriox2]@mercurio &lt;all-applications&gt;"></incidentoverviewrecord>
          </incidentoverviewrecord>
        </incidentoverviewrecord>
      </incidentoverviewrecords>
    </incidentsoverviewdashlet>
  </data>
</dashboardreport>
] 
[26 feb 2018 16:29:29,307][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_ANAGRAFICA] (ARAPPX02_QM_THRESHOULD_10@192.168.175.144) ARAPPX02_QM_THRESHOULD_10 upper bound exceeded" start="2018-02-12T19:44:10.000+01:00" source="ARAPPX02_QM_THRESHOULD_10@192.168.175.144"/>] 
[26 feb 2018 16:29:29,308][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_NAS] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T19:53:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"/>] 
[26 feb 2018 16:29:29,309][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_UFIS_ADR] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T17:30:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"/>] 
[26 feb 2018 16:29:29,310][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_UFIS_TEST] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T17:30:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"/>] 
[26 feb 2018 16:29:29,310][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->TELEX_QU_ADBM] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded" start="2018-02-23T14:08:10.000+01:00" source="ARAPPX02_QM - Test1@192.168.175.144"/>] 
[26 feb 2018 16:29:29,311][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Java Virtual Machine/Total GC Collection Time: Total GC Collection Time &lt;all-applications> (WebSphere_ADBM_TEST[arappx03Cell01-app_adbm_mercuriox2]@mercurio) upper bound exceeded" start="2018-02-26T03:00:10.000+01:00" duration="10000" end="2018-02-26T03:00:20.000+01:00" source="WebSphere_ADBM_TEST[arappx03Cell01-app_adbm_mercuriox2]@mercurio &lt;all-applications>"/>] 
[26 feb 2018 16:29:29,458][DEBUG][AC1F07675A9427D70001B514][1] name:[Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_UFIS_TEST] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded startEvent:[2018-02-23 17:30:10.0] 
[26 feb 2018 16:29:29,461][DEBUG][AC1F07675A9427D70001B514][1] Incident alreay Exists:[DynatraceIncidentKey [name=Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_UFIS_TEST] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded, startEvent=2018-02-23 17:30:10.0]] update the DateUpdate 
[26 feb 2018 16:29:29,475][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,484][DEBUG][AC1F07675A9427D70001B514][1] name:[Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->TELEX_QU_ADBM] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded startEvent:[2018-02-23 14:08:10.0] 
[26 feb 2018 16:29:29,485][DEBUG][AC1F07675A9427D70001B514][1] Incident alreay Exists:[DynatraceIncidentKey [name=Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->TELEX_QU_ADBM] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded, startEvent=2018-02-23 14:08:10.0]] update the DateUpdate 
[26 feb 2018 16:29:29,496][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,507][DEBUG][AC1F07675A9427D70001B514][1] name:[Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_NAS] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded startEvent:[2018-02-23 19:53:10.0] 
[26 feb 2018 16:29:29,507][DEBUG][AC1F07675A9427D70001B514][1] Incident alreay Exists:[DynatraceIncidentKey [name=Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_NAS] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded, startEvent=2018-02-23 19:53:10.0]] update the DateUpdate 
[26 feb 2018 16:29:29,518][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,529][DEBUG][AC1F07675A9427D70001B514][1] name:[Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_ANAGRAFICA] (ARAPPX02_QM_THRESHOULD_10@192.168.175.144) ARAPPX02_QM_THRESHOULD_10 upper bound exceeded startEvent:[2018-02-12 19:44:10.0] 
[26 feb 2018 16:29:29,530][DEBUG][AC1F07675A9427D70001B514][1] Incident alreay Exists:[DynatraceIncidentKey [name=Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_ANAGRAFICA] (ARAPPX02_QM_THRESHOULD_10@192.168.175.144) ARAPPX02_QM_THRESHOULD_10 upper bound exceeded, startEvent=2018-02-12 19:44:10.0]] update the DateUpdate 
[26 feb 2018 16:29:29,539][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,548][DEBUG][AC1F07675A9427D70001B514][1] name:[Java Virtual Machine/Total GC Collection Time: Total GC Collection Time <all-applications> (WebSphere_ADBM_TEST[arappx03Cell01-app_adbm_mercuriox2]@mercurio) upper bound exceeded startEvent:[2018-02-26 03:00:10.0] 
[26 feb 2018 16:29:29,549][DEBUG][AC1F07675A9427D70001B514][1] Incident alreay Exists:[DynatraceIncidentKey [name=Java Virtual Machine/Total GC Collection Time: Total GC Collection Time <all-applications> (WebSphere_ADBM_TEST[arappx03Cell01-app_adbm_mercuriox2]@mercurio) upper bound exceeded, startEvent=2018-02-26 03:00:10.0]] update the DateUpdate 
[26 feb 2018 16:29:29,561][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,569][DEBUG][AC1F07675A9427D70001B514][1] name:[Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_UFIS_ADR] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded startEvent:[2018-02-23 17:30:10.0] 
[26 feb 2018 16:29:29,569][DEBUG][AC1F07675A9427D70001B514][1] Incident alreay Exists:[DynatraceIncidentKey [name=Queue Group/CURRENT_Q_DEPTH: CURRENT_Q_DEPTH [Queue Name->ADBM_QU_UFIS_ADR] (ARAPPX02_QM - Test1@192.168.175.144) ARAPPX02_QM - Test1 upper bound exceeded, startEvent=2018-02-23 17:30:10.0]] update the DateUpdate 
[26 feb 2018 16:29:29,577][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,577][DEBUG][AC1F07675A9427D70001B514][1] the Name for app:[TAXI] is [TAXI] 
[26 feb 2018 16:29:29,577][DEBUG][AC1F07675A9427D70001B514][1] the URL for app:[TAXI] is [https://dynatrace-sub-ca:8021/rest/management/dashboard/TAXI_TEST_REMEDY_INTEGRATION] 
[26 feb 2018 16:29:29,577][DEBUG][AC1F07675A9427D70001B514][1] the USR for app:[TAXI] is [admin] 
[26 feb 2018 16:29:29,578][DEBUG][AC1F07675A9427D70001B514][1] the PWD for app:[TAXI] is [adradmin01] 
[26 feb 2018 16:29:29,578][DEBUG][AC1F07675A9427D70001B514][1] calling the console:[https://dynatrace-sub-ca:8021/rest/management/dashboard/TAXI_TEST_REMEDY_INTEGRATION] with user[admin] pwd=adradmin01... 
[26 feb 2018 16:29:29,578][DEBUG][AC1F07675A9427D70001B514][1] Executing POST with basic authentication 
[26 feb 2018 16:29:29,578][DEBUG][AC1F07675A9427D70001B514][1] SOAP HTTP request headers: 
[26 feb 2018 16:29:29,578][INFO ][AC1F07675A9427D70001B514][1] 	name=Authorization, value=Basic YWRtaW46YWRyYWRtaW4wMQ== 
[26 feb 2018 16:29:29,896][INFO ][AC1F07675A9427D70001B514][1] Remote peer HTTP version > 1.0 --> keep-alive=true 
[26 feb 2018 16:29:29,897][INFO ][AC1F07675A9427D70001B514][1] Current connection will be kept alive 'indefinitely' 
[26 feb 2018 16:29:29,897][INFO ][AC1F07675A9427D70001B514][1] SOAP HTTP response headers: 
[26 feb 2018 16:29:29,897][INFO ][AC1F07675A9427D70001B514][1] 	name=Date, value=Mon, 26 Feb 2018 15:29:29 GMT 
[26 feb 2018 16:29:29,898][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Disposition, value=inline; filename="TAXI_TEST_REMEDY_INTEGRATION.xml" 
[26 feb 2018 16:29:29,898][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Type, value=text/xml 
[26 feb 2018 16:29:29,898][INFO ][AC1F07675A9427D70001B514][1] 	name=Cache-Control, value=no-cache 
[26 feb 2018 16:29:29,899][INFO ][AC1F07675A9427D70001B514][1] 	name=Cache-Control, value=must-revalidate 
[26 feb 2018 16:29:29,899][INFO ][AC1F07675A9427D70001B514][1] 	name=Cache-Control, value=max-age=0 
[26 feb 2018 16:29:29,900][INFO ][AC1F07675A9427D70001B514][1] 	name=Pragma, value=no-cache 
[26 feb 2018 16:29:29,900][INFO ][AC1F07675A9427D70001B514][1] 	name=Vary, value=Accept-Encoding, User-Agent 
[26 feb 2018 16:29:29,900][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Length, value=2288 
[26 feb 2018 16:29:29,901][INFO ][AC1F07675A9427D70001B514][1] HTTP response code received: 200 
[26 feb 2018 16:29:29,901][DEBUG][AC1F07675A9427D70001B514][1] calling the console:[https://dynatrace-sub-ca:8021/rest/management/dashboard/TAXI_TEST_REMEDY_INTEGRATION] with user[admin] pwd=adradmin01... DONE elapsed:[323] 
[26 feb 2018 16:29:29,902][DEBUG][AC1F07675A9427D70001B514][1] content:[<?xml version="1.0" encoding="utf-8"?>
<dashboardreport name="TAXI_TEST_REMEDY_INTEGRATION" version="7.0.10.1010" reportdate="2018-02-26T16:29:30.006+01:00" description="">
  <source name="TAXI_TEST">
    <filters summary="last 24 hours">
      <filter>tf:Last24h</filter>
    </filters>
  </source>
  <data>
    <incidentsoverviewdashlet name="Incidents" description="" displaysource="Base">
      <source>
        <filters summary="Incident Rules, last 24 hours">
          <filter>if:RULE?Agent Instrumentation Disabledif:RULE?Application Process Out-of-Memoryif:RULE?Application Process Unavailable (unexpected)if:RULE?Application Process Unhealthyif:RULE?Average response time degradedif:RULE?Database connection issueif:RULE?Failure rate too highif:RULE?Host Availabilityif:RULE?Host CPU Unhealthyif:RULE?Host Disk Unhealthyif:RULE?Host Memory Unhealthyif:RULE?Host Network Unhealthyif:RULE?Response time degraded for slow requests</filter>
          <filter>tf:Last24h</filter>
        </filters>
      </source>
      <incidentoverviewrecords structuretype="tree">
        <incidentoverviewrecord name="TAXI_TEST (1)">
          <incidentoverviewrecord name="Agent Instrumentation Disabled"></incidentoverviewrecord>
          <incidentoverviewrecord name="Application Process Out-of-Memory"></incidentoverviewrecord>
          <incidentoverviewrecord name="Application Process Unavailable (unexpected) (1)">
            <incidentoverviewrecord name="Agent 'CIAJVM-PMS_TAXI_TEST@adrcdc05' connection lost" start="2018-02-26T01:05:43.721+01:00" duration="16106" end="2018-02-26T01:05:59.827+01:00" source="-"></incidentoverviewrecord>
          </incidentoverviewrecord>
          <incidentoverviewrecord name="Application Process Unhealthy"></incidentoverviewrecord>
          <incidentoverviewrecord name="Database Agent Connection Issue"></incidentoverviewrecord>
          <incidentoverviewrecord name="Failure rate too high"></incidentoverviewrecord>
          <incidentoverviewrecord name="Response time degraded"></incidentoverviewrecord>
          <incidentoverviewrecord name="Response time degraded for slow requests"></incidentoverviewrecord>
        </incidentoverviewrecord>
      </incidentoverviewrecords>
    </incidentsoverviewdashlet>
  </data>
</dashboardreport>
] 
[26 feb 2018 16:29:29,921][DEBUG][AC1F07675A9427D70001B514][1] ------ALLARME-----:[<xml-fragment name="Agent 'CIAJVM-PMS_TAXI_TEST@adrcdc05' connection lost" start="2018-02-26T01:05:43.721+01:00" duration="16106" end="2018-02-26T01:05:59.827+01:00" source="-"/>] 
[26 feb 2018 16:29:29,950][DEBUG][AC1F07675A9427D70001B514][1] Updated 1 rows 
[26 feb 2018 16:29:29,950][DEBUG][AC1F07675A9427D70001B514][1] Incident inserted:[DynatraceIncidentKey [name=Agent 'CIAJVM-PMS_TAXI_TEST@adrcdc05' connection lost, startEvent=2018-02-26 01:05:43.721] 
[26 feb 2018 16:29:29,951][INFO ][AC1F07675A9427D70001B514][1] Service:[it.reply.sytel.adr.services.impl.GetDynatraceIncident] called ... DONE with ServiceExecutionTime:[2604] 
[26 feb 2018 16:29:29,951][INFO ][AC1F07675A9427D70001B514][1] service:[it.reply.sytel.adr.services.impl.CreateRemedyIncident] called ...  
[26 feb 2018 16:29:29,966][INFO ][AC1F07675A9427D70001B514][1] Number of incident to create on Remedy:[7] 
[26 feb 2018 16:29:30,197][DEBUG][AC1F07675A9427D70001B514][1] Exception on helpDeskSubmitServiceDocument:[<urn:HelpDesk_Submit_Service xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP"><urn:First_Name xsi:nil="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Impact>1-Extensive/Widespread</urn:Impact><urn:Last_Name xsi:nil="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Reported_Source>Systems Management</urn:Reported_Source><urn:Service_Type>Infrastructure Event</urn:Service_Type><urn:Status xsi:nil="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Summary>.........SUMNMARY..........</urn:Summary><urn:Urgency>1-Critical</urn:Urgency></urn:HelpDesk_Submit_Service>] 
[26 feb 2018 16:29:30,241][INFO ][AC1F07675A9427D70001B514][1] EnvelopeToSend sent to Remedy:[<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <Header>
    <urn:AuthenticationInfo>
      <urn:userName>RemedyUsr</urn:userName>
      <urn:password>RemedyPwd</urn:password>
    </urn:AuthenticationInfo>
  </Header>
  <Body>
    <urn:HelpDesk_Submit_Service>
      <urn:First_Name xsi:nil="true"/>
      <urn:Impact>1-Extensive/Widespread</urn:Impact>
      <urn:Last_Name xsi:nil="true"/>
      <urn:Reported_Source>Systems Management</urn:Reported_Source>
      <urn:Service_Type>Infrastructure Event</urn:Service_Type>
      <urn:Status xsi:nil="true"/>
      <urn:Summary>.........SUMNMARY..........</urn:Summary>
      <urn:Urgency>1-Critical</urn:Urgency>
    </urn:HelpDesk_Submit_Service>
  </Body>
</Envelope>] 
[26 feb 2018 16:29:30,245][INFO ][AC1F07675A9427D70001B514][1] Executing POST request to URL: POST http://syt12303:8088/mockHPD_IncidentInterface_Create_WSPortTypeSoapBinding HTTP/1.1 
[26 feb 2018 16:29:30,256][INFO ][AC1F07675A9427D70001B514][1] Remote peer HTTP version > 1.0 --> keep-alive=true 
[26 feb 2018 16:29:30,256][INFO ][AC1F07675A9427D70001B514][1] Current connection will be kept alive 'indefinitely' 
[26 feb 2018 16:29:30,256][INFO ][AC1F07675A9427D70001B514][1] SOAP HTTP response headers: 
[26 feb 2018 16:29:30,257][INFO ][AC1F07675A9427D70001B514][1] 	name=Content-Type, value=text/html; charset=iso-8859-1 
[26 feb 2018 16:29:30,257][INFO ][AC1F07675A9427D70001B514][1] 	name=Transfer-Encoding, value=chunked 
[26 feb 2018 16:29:30,257][INFO ][AC1F07675A9427D70001B514][1] 	name=Server, value=Jetty(6.1.26) 
[26 feb 2018 16:29:30,257][INFO ][AC1F07675A9427D70001B514][1] HTTP response code received: 200 
[26 feb 2018 16:29:30,265][INFO ][AC1F07675A9427D70001B514][1] Return from Remedy :[<html><body><p>There are currently 0 running SoapUI MockServices</p><ul></ul></p></body></html>] 
[26 feb 2018 16:29:30,265][INFO ][AC1F07675A9427D70001B514][1] Starting gettting Remedy Response 
[26 feb 2018 16:29:30,290][ERROR][AC1F07675A9427D70001B514][1] Exception on Flow Flow TestBuilder Engine 
it.reply.sytel.adr.services.exc.CreateRemedyIncidentException: Exception on CreateRemedyIncident; nested exception is: 
	it.reply.sytel.adr.remedyAdapter.exc.RemedyWSClientException: Exception on calling Remedy WS. CreateInputMap:[<xml-fragment><urn:First_Name xsi:nil="true" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Impact xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">1-Extensive/Widespread</urn:Impact><urn:Last_Name xsi:nil="true" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Reported_Source xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">Systems Management</urn:Reported_Source><urn:Service_Type xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">Infrastructure Event</urn:Service_Type><urn:Status xsi:nil="true" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Summary xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">.........SUMNMARY..........</urn:Summary><urn:Urgency xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">1-Critical</urn:Urgency></xml-fragment>]; nested exception is: 
	it.reply.sytel.adr.remedyAdapter.exc.RemedyWSClientException: Exception on getting the RemedyResponse; nested exception is: 
	org.apache.xmlbeans.XmlException: error: </p> does not close tag <body>.
it.reply.sytel.adr.remedyAdapter.exc.RemedyWSClientException: Exception on calling Remedy WS. CreateInputMap:[<xml-fragment><urn:First_Name xsi:nil="true" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Impact xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">1-Extensive/Widespread</urn:Impact><urn:Last_Name xsi:nil="true" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Reported_Source xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">Systems Management</urn:Reported_Source><urn:Service_Type xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">Infrastructure Event</urn:Service_Type><urn:Status xsi:nil="true" xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/><urn:Summary xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">.........SUMNMARY..........</urn:Summary><urn:Urgency xmlns:urn="urn:HPD_IncidentInterface_Create_WS_VIP">1-Critical</urn:Urgency></xml-fragment>]; nested exception is: 
	it.reply.sytel.adr.remedyAdapter.exc.RemedyWSClientException: Exception on getting the RemedyResponse; nested exception is: 
	org.apache.xmlbeans.XmlException: error: </p> does not close tag <body>.
it.reply.sytel.adr.remedyAdapter.exc.RemedyWSClientException: Exception on getting the RemedyResponse; nested exception is: 
	org.apache.xmlbeans.XmlException: error: </p> does not close tag <body>.
org.apache.xmlbeans.XmlException: error: </p> does not close tag <body>.
	at org.apache.xmlbeans.impl.store.Locale$SaxLoader.load(Locale.java:3486)
	at org.apache.xmlbeans.impl.store.Locale.parseToXmlObject(Locale.java:1276)
	at org.apache.xmlbeans.impl.store.Locale.parseToXmlObject(Locale.java:1263)
	at org.apache.xmlbeans.impl.schema.SchemaTypeLoaderBase.parse(SchemaTypeLoaderBase.java:345)
	at org.xmlsoap.schemas.soap.envelope.EnvelopeDocument$Factory.parse(EnvelopeDocument.java:71)
	at it.reply.sytel.adr.remedyAdapter.impl.RemedyWSClientImpl.getResponse(RemedyWSClientImpl.java:153)
	at it.reply.sytel.adr.remedyAdapter.impl.RemedyWSClientImpl.callRemedy(RemedyWSClientImpl.java:141)
	at it.reply.sytel.adr.remedyAdapter.impl.RemedyWSClientImpl.createIncident(RemedyWSClientImpl.java:119)
	at it.reply.sytel.adr.services.impl.CreateRemedyIncident.perform(CreateRemedyIncident.java:60)
	at it.reply.sytel.adr.core.services.service.AbstractService.execute(AbstractService.java:66)
	at it.reply.sytel.adr.engine.RemedyIncidetBuilderEngine.executeFlow(RemedyIncidetBuilderEngine.java:60)
	at it.reply.sytel.adr.main.TestMain.main(TestMain.java:30)
Caused by: org.xml.sax.SAXParseException; systemId: file:; lineNumber: 1; columnNumber: 80; </p> does not close tag <body>.
	at org.apache.xmlbeans.impl.piccolo.xml.Piccolo.reportFatalError(Piccolo.java:1038)
	at org.apache.xmlbeans.impl.piccolo.xml.Piccolo.parse(Piccolo.java:723)
	at org.apache.xmlbeans.impl.store.Locale$SaxLoader.load(Locale.java:3454)
	... 11 more
